metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/EnrichmentWorkerImplTest.java

```diff
@@ -34,10 +34,12 @@
 import eu.europeana.enrichment.rest.client.report.Type;
```

Comment:
Oh and one more thing.
I've put a break point inside the new aggregation method and there seems to be no tests reaching that point.